core/server-entry.ts

```diff
@@ -43,6 +44,10 @@ function _ssrHydrateSubcomponents (components, store, router, resolve, reject, a
         if (excludeFromConfig && excludeFromConfig.length > 0) {
           context.state.config = omit(context.state.config, excludeFromConfig)
         }
+        context.state.config = purgeConfigMethod({
```

Comment:
`purgeConfigMethod` returns string and `context.state.config` should be still an object here :thinking: